collector/perfos.go

```diff
@@ -0,0 +1,133 @@
+// returns data points from Win32_PerfRawData_PerfOS_System class (undocumented)
+
+package collector
+
+import (
+	"log"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	Factories["perfos"] = NewPerfOSCollector
+}
+
+// A PerfOSCollector is a Prometheus collector for WMI metrics
+type PerfOSCollector struct {
+	// XXX names
+	ContextSwitchesPersec     *prometheus.Desc
```

Comment:
Are these "per second" metrics actually per second? The ones in `logical_disk` were actually counters